apps/emqx_rule_engine/src/emqx_rule_runtime.erl

```diff
@@ -523,9 +525,7 @@ inc_action_metrics(R, RuleId) ->
             emqx_metrics_worker:inc(rule_metrics, RuleId, 'actions.success')
     end.
 
-is_ok_result(ok) ->
```

Comment:
`ok` is already matched on line 511:
```
inc_action_metrics(ok, RuleId) ->
    emqx_metrics_worker:inc(rule_metrics, RuleId, 'actions.success');
```

Reply:
if ok results has to be leaked down here anyway we can probably delete line 511 instead.